uploadNeuroDB/minc_insertion.pl

```diff
@@ -455,6 +465,40 @@
     NeuroDB::MRI::make_nii(\$file, $data_dir);
 }
 
+############################################################
+############# Create minc-pics #############################
+############################################################
+if ($create_minc_pics)
```

Comment:
@gluneau,

this can be taken outside to a utility file, this way tarchiveLoader and minc_insertion can reuse the same function